main.go

```diff
@@ -541,17 +542,17 @@ is available in the $BITRISE_IDEDISTRIBUTION_LOGS_PATH environment variable`)
 				deployPth := filepath.Join(configs.DeployDir, base)
 
 				if err := command.CopyFile(ipa, deployPth); err != nil {
-					fail("Failed to copy (%s) -> (%s), error: %s", ipas[0], ipaPath, err)
+					fail("Failed to copy (%s) -> (%s), error: %s", ipas[0], exportedIPAPath, err)
```

Comment:
exportedIPAPath is empty if logged here, you meant "ipa"?

Reply:
👍 I will create a new PR for this.